lib/datadog.py

```diff
@@ -305,10 +305,14 @@ def compile(install_path, cache_dir):
     download(install_path, cache_dir)
 
 
-def run():
+def run(runtime_version):
     if not is_enabled():
         return
 
+    if runtime_version < 7.14:
+        logger.error("Datadog integration requires Mendix 7.14 or newer")
+        raise Exception("Datadog integration requires Mendix 7.14 or newer")
+
```

Comment:
Let's check this in the compilation phase of the build pack, and not during run. I'm also not sure an exception is needed stopping the app from running or just a big warning in the log.